site/it/community/contribute/code_style.md

```diff
@@ -0,0 +1,164 @@
+# Guida allo stile del codice TensorFlow
+
+## Stile Python
+
+Segue la guida [PEP 8 Python style
+](https://www.python.org/dev/peps/pep-0008/), ad eccezione, per TensorFlow, di usare 2
+spazi invece di 4. Cortesemente, allineatevi alla
+[Guida Google di Stile per Python](https://github.com/google/styleguide/blob/gh-pages/pyguide.md),
+ed usate [pylint](https://www.pylint.org/) per controllare le vostre modifiche Python.
+
+
+### pylint
+
+Per installare `pylint` e recuperare le definizioni personalizzate di stile di TensorFlow:
+
+```bash
+
+$ pip install pylint
+$ wget -O /tmp/pylintrc https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/tools/ci_build/pylintrc
+
+```
+
+Per controllare un file con `pylint`:
+
+```bash
+$ pylint --rcfile=/tmp/pylintrc myfile.py
+```
+
+### Versioni di Python supportate
+
+TensorFlow supporta Python 2.7 e Python >= 3.4. Vedere la
+[guida di installazione](https://www.tensorflow.org/install) per i dettagli.
+
+Vedere lo 
+[stato del build continuo](https://github.com/tensorflow/tensorflow/blob/master/README.md#continuous-build-status)
```

Comment:
It used "integrazione continua" but more frequent "continous integration/CI".  
Some use deploy continuo that is in the same mixed EN-IT style of "build continuo":
https://www.redhat.com/it/topics/devops/what-is-ci-cd

Reply:
In my experience there is not a common practice for "continuous integration" / "continuous development" / "continuous deployment" or "continuous building" translation. 
In my opinion a good translation should minimise not-translated terms. On this way, we are forced to leave "build/building" in English, because this word has a very peculiar meaning in IT context, related with the transformation of source code in executable code. On the other hand, I've chose to translate "continuous" because there is no peculiar meaning related with "continuous build". It simply means: "to build continuously"... In any case, I'm available for any update...